### PR TITLE
Implement basic functionality for crypto stats

### PR DESCRIPTION
**Pull request by jbaldwinroberts**
```
$ ./rust-crypto-tracker -c bitcoin -f "{symbol} \${price_usd} {percent_change_24h}%"
BTC $2749.82 1.17%
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
 # rust-crypto-tracker
 Small command line tool to track crypto stocks
+
+```
+$ ./target/release/rust-crypto-tracker -h
```
Comment (CameronDiver) [inline README.md]: This should be just the executable name.

File: rustfmt.toml [hunk 2]
```diff
@@ -9,3 +9,4 @@ match_block_trailing_comma = true
 struct_lit_multiline_style = "ForceMulti"
 struct_lit_style = "Block"
 use_try_shorthand = true
+max_width = 1024
```
Comment (CameronDiver) [inline rustfmt.toml]: This shouldn't be here.

File: src/main.rs [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        )
+        .arg(
+            Arg::with_name("format")
+                .short("f")
+                .long("format")
+                .help("Enter the format")
+                .long_help("longer")
+                .takes_value(true)
+                .default_value(
+                    "{name}, {symbol}, {rank}, {price_usd}, {price_btc}, {24h_volume_usd}, {market_cap_usd}, {available_supply}, {total_supply}, {percent_change_1h}, {percent_change_24h}, {percent_change_7d}, {last_updated}",
```
Comment (CameronDiver) [inline src/main.rs]: I think if you just have `"several" "string" "literals"` it should be fine.

File: src/main.rs [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                ),
+        )
+        .get_matches();
+
+    let mut url = PathBuf::from("http://api.coinmarketcap.com/v1/ticker/");
+    url.push(matches.value_of("crypto").unwrap());
+
+    let client = Client::new();
+    let mut response = client.get(url.to_str().unwrap()).send().unwrap();
```
Comment (CameronDiver) [inline src/main.rs:50]: We should make an issue to address the error handling here.

**Review by CameronDiver — CHANGES_REQUESTED** [review]

File: src/main.rs (hunk 4 above)
Comment (jbaldwinroberts) [reply src/main.rs:50]: #1

**Review by CameronDiver — APPROVED** [review]

**Pull request merged**